src/main/java/net/sf/jabref/gui/FXAlert.java

```diff
@@ -89,6 +95,14 @@ public FXAlert(AlertType type) {
         fxDialogWindow.setOnHiding(evt -> setSwingWindowsEnabledAndFocusable(true));
 
         fxDialogWindow.setOnCloseRequest(evt -> this.close());
+
+        fxDialogWindow.getScene().setOnKeyPressed(evt -> {
+            KeyBindingPreferences keyPreferences = Globals.getKeyPrefs();
+            KeyCombination combination = keyPreferences.getKeyCombination(KeyBinding.CLOSE_DIALOG);
+            if (keyPreferences.checkKeyCombinationEquality(combination, evt)) {
```

Comment:
Can you please also introduce an overload so that you can simply write `if keyPreferences.checkKeyCombinationEquality(KeyBinding.CLOSE_DIALOG, evt)`.